instructions/instructions.css

```diff
@@ -0,0 +1,392 @@
+html {
+  scroll-snap-type: x mandatory;
+  /* to swipe between pages*/
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: var(--primary-color);
+  /* FOR DARK THEME */
+  font-family: IBM Plex Sans;
+  background-repeat: no-repeat;
+  background-size: cover;
```

Comment:
Why did you use this feature here you need it when adding an image as background